Plugins/DeleteUnusedBranches/DataGridViewCheckBoxHeaderCell.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using System.Windows.Forms.VisualStyles;
+
+namespace DeleteUnusedBranches
+{
+    public class DataGridViewCheckBoxHeaderCell : DataGridViewColumnHeaderCell
+    {
+        private Point _checkBoxLocation;
+        private Size _checkBoxSize;
+        private Point _cellLocation = new Point();
+        private CheckBoxState _cbState = CheckBoxState.UncheckedNormal;
```

Comment:
no need to initialise, it is being set before use on line 57.
in fact I don't see any other uses of it, any reason it is not a local variable?